src/components/trash-page/trash-page.module.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/common/variables.module.scss';
+
+.root {
+    padding: $size24 $size32;
+}
+
+.notes-list {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $size16;
+}
+
+.searchInput {
+    margin: 0px auto $size2;
+}
+.scrollable-area {
+    width: 100%;
+    height: 100vh;
```

Comment:
Is is a correct style?
Scroll area in your case doesn't take the whole viewport

Reply:
I've put 600px so far. Ideally I should have calculate height as viewport height - search height - section title height. 

Maybe I can do it better with flex, not sure

Reply:
Yes. It can be done with flex or grid